code/2_analysis/maps/fig3a.R

```diff
@@ -0,0 +1,158 @@
+# CLEAN VIRTUAL WATER DATA -----------------------------------------------------
```

Comment:
this script is a bit more than a standard plotting script like the other maps figures. i think a slightly more detailed header would be helpful.

Reply:
Agreed! Adding some more detail!